css/style.css

```
@@ -154,15 +221,26 @@ ul {
 }
 
 .contact-data {
+    position: relative;
     flex-basis: 270px;
 }
 
+.contact-data a::before {
+    content: "";
+    position: absolute;
+    background-image: url(../img/icon-phone.svg);
+    width: 19px;
+    height: 19px;
+    top: 13px;
+    left: 39px;
+}
```

Если контент псевдоэлемента - картинка, то можно ее указывать напрямую в правиле `content`
```suggestion
.contact-data a::before {
    content: url(../img/icon-phone.svg);
    position: absolute;
    top: 13px;
    left: 39px;
}
```